docs/appendices/release-notes/5.8.1.rst

```diff
@@ -47,6 +47,13 @@ See the :ref:`version_5.8.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Changed behavior for partitioned tables by automatically calculating
+  :ref:`number_of_routing_shards <sql-create-table-number-of-routing-shards>`,
+  so that they behave the same as simple tables, for which the
+  :ref:`change<version_5.8.0_break_ch_no_route_shards>` was already introduced
+  in :ref:`version_5.8.0`.
```

Comment:
Shouldn't we focus on a single partition case here which was the real problem?

Reply:
Adjusted, please check again, thx!

Reply:
LGTM, thanks. 

>will wait for https://github.com/crate/crate/pull/16412 to be merged first and adjust the integration test.

+1, https://github.com/crate/crate/pull/16412 should come first